compiler/src/iree/compiler/Codegen/Common/GPU/GPUTileSwizzleUtils.h

```diff
@@ -27,7 +33,8 @@ TileSwizzle getIntrinsicSwizzle(IREE::GPU::MMAIntrinsic intrinsic,
 //    Input unrollFactor = 4
 // -> Output swizzle = { expandShape = [[16], [4, 4]], permutation = [1, 2, 0] }
 //
-void unroll(TileSwizzle &swizzle, int srcIndex, int unrollFactor);
+void unroll(TileSwizzle &swizzle, int srcIndex, int unrollFactor,
+            bool cross_thread, bool cross_intrinsic);
```

Comment:
Add a short explanation of the new fields in the doc.

Reply:
done.